apps/api/src/modules/etl/event-plans/delete.ts

```diff
@@ -0,0 +1,67 @@
+import assert from 'assert';
+import Debug from 'debug';
+import { App } from 'firebase-admin/app';
+import { getAuth as getFirebaseAuth } from 'firebase-admin/auth';
+import { getFirestore as getFirebaseFirestore } from 'firebase-admin/firestore';
+
+import { EventPlanId, UserId } from '../../../interfaces';
+import { ApiError, makeApiError } from '../../../../lib/errors';
+
+type Params = {
+  eventPlanId: EventPlanId;
+  invitees: UserId[];
```

Comment:
Same note here -> inviteesByUserId is arguably a better name

Better to lean on best practices over language features. For example better to give a better variable name than to rely on Typescript's type features. We should use Typescript's type to help support our code rather than the other way around.

Reply:
Sorry just realizing it now but I don't even think we need to pass in an array of invitees.

We really only need to pass in the eventPlanId and then maybe the userId (userId of whoever is making the call so we can hackily check if they are the host or not).

Use a transaction if you can to fetch the event-plan doc corresponding to the params.eventPlanId and thats where we get the inviteesByUserId from.